public/app/features/alerting/unified/mocks/server/handlers/k8s/receivers.k8s.ts

```diff
@@ -20,7 +21,12 @@ const mappedReceivers =
       metadata: {
         // This isn't exactly accurate, but its the cleanest way to use the same data for AM config and K8S responses
         uid: camelCase(contactPoint.name),
-        annotations: { [PROVENANCE_ANNOTATION]: provenance },
+        annotations: {
```

Comment:
TODO as well (maybe in this PR, maybe not, thoughts?) - a cleaner way to manage entities here and allow for contact points that have different permutations of permissions, and inUse information - e.g ones that are in use by alert rules but not policies and vice versa